src/lib/data/filterables.ts

```diff
@@ -803,6 +858,11 @@ export const filterableTypes: Filterable[] = [
 		aliases: ['diango', 'daily', 'dailies'],
 		items: diango
 	},
+	{
+		name: 'Diaries',
+		aliases: ['diaries', 'diary', 'ad'],
+		items: diaries
+	},
```

Comment:
test it!

Reply:
![image](https://user-images.githubusercontent.com/5985728/122806175-4f43c400-d2ca-11eb-9024-dbcd53b9f083.png)

Reply:
I totally misread what this was for, i thought i was looking at these being added to the CL.